.github/workflows/build-test.yml

```diff
@@ -1,6 +1,12 @@
 name: Build, Test
-
+run-name: CI check
```

Comment:
why is this added? now "Build, Test" is changed to "CI check"

https://github.com/OffchainLabs/arbitrum-token-bridge/actions/workflows/build-test.yml